app.js

```diff
@@ -30,12 +31,47 @@
       this.$('.query.date').datepicker({ dateFormat: "yy-mm-dd" });
     }),
 
+    // Toggles acending/decending order of the column header clicked
+    sortTable: function(event) {
+      var $tr = this.$(event.target);
+      var position = $tr.index();
+      var $tableBody = $tr.closest('table').find('tbody');
+      var greaterThan,
+          lessThan;
+
+      $tr.addClass('sorted');
+      $tr.siblings().removeClass('sorted ascending');
+      $tr.toggleClass('ascending');
+
+      if ( $tr.hasClass('ascending') ) {
+        greaterThan = 1;
+        lessThan = -1;
+      } else {
+        greaterThan = -1;
+        lessThan = 1;
+      }
+
+      var newList = $tableBody.find('tr').sort( function(a,b) {
+        if ( this.$(a).find('td:eq(' + position + ')').text() > this.$(b).find('td:eq(' + position + ')').text() ){
```

Comment:
I wrote a note here, but it looks like it didn't post. Unfortunately, `string - string` doesn't return a numerical value, it returns `NaN`:
```javascript
"this" > "that" // returns true
"this" - "that" // returns NaN
```

`>` and `<` evaluate the unicode value for the characters in the strings. I actually have no idea what `-` does to strings, but I assume that it doesn't return a number because the converse operator (`+`) concatenates strings and doesn't add their unicode values together.